design/design-doc-stacks.md

```diff
@@ -8,7 +8,7 @@ This document aims to provide details about the experience and implementation fo
 ## Revisions
 
 * 1.1
-  * Renamed Extensions concept to Stacks (`Extension` code references are unaffected) [#571](https://github.com/crossplaneio/crossplane/issues/571)
+  * Renamed Stacks concept to Stacks (`Stack` code references are unaffected) [#571](https://github.com/crossplaneio/crossplane/issues/571)
```

Comment:
Do you want to keep this as "Renamed *Extensions* concept to Stacks"? Find and replace will get ya :)

Reply:
Fixed in https://github.com/crossplaneio/crossplane/pull/679/files#diff-5db872f8c3bbf22fd1da26bc1a5c1f30R11 and subsequent squashes